Import/PaymentMethodImporter.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdyenPayment\Import;
+
+use AdyenPayment\Components\Adyen\Mapper\PaymentMethodMapper;
+use AdyenPayment\Components\Adyen\PaymentMethod\PaymentMethodsProvider;
+use AdyenPayment\Doctrine\Writer\PaymentMethodWriter;
+use AdyenPayment\Models\PaymentMethod\ImportResult;
+use AdyenPayment\Rule\AdyenApi\UsedFallbackConfigRule;
+use Doctrine\Common\Persistence\ObjectRepository;
+use Shopware\Components\Model\ModelManager;
+use Shopware\Models\Shop\Shop;
+
+class PaymentMethodImporter implements PaymentMethodImporterInterface
+{
+    /**
+     * @var PaymentMethodsProvider
+     */
+    private $paymentMethodsProvider;
+    /**
+     * @var ObjectRepository
+     */
+    private $shopRepository;
+    /**
+     * @var UsedFallbackConfigRule
+     */
+    private $usedFallbackConfigRule;
+    /**
+     * @var PaymentMethodMapper
+     */
+    private $paymentMethodMapper;
+    /**
+     * @var PaymentMethodWriter
+     */
+    private $paymentMethodWriter;
+    /** @var ModelManager */
+    private $entityManager;
+
+    public function __construct(
+        PaymentMethodsProvider $paymentMethodsProvider,
+        ObjectRepository $shopRepository,
+        UsedFallbackConfigRule $usedFallbackConfigRule,
+        PaymentMethodMapper $paymentMethodMapper,
+        PaymentMethodWriter $paymentMethodWriter,
+        ModelManager $entityManager
+    )
+    {
+        $this->paymentMethodsProvider = $paymentMethodsProvider;
+        $this->shopRepository = $shopRepository;
+        $this->usedFallbackConfigRule = $usedFallbackConfigRule;
+        $this->paymentMethodMapper = $paymentMethodMapper;
+        $this->paymentMethodWriter = $paymentMethodWriter;
+        $this->entityManager = $entityManager;
+
+    }
```

Comment:
code style clean-up:

```suggestion
    public function __construct(
        PaymentMethodsProvider $paymentMethodsProvider,
        ObjectRepository $shopRepository,
        UsedFallbackConfigRule $usedFallbackConfigRule,
        PaymentMethodMapper $paymentMethodMapper,
        PaymentMethodWriter $paymentMethodWriter,
        ModelManager $entityManager
    ) {
        $this->paymentMethodsProvider = $paymentMethodsProvider;
        $this->shopRepository = $shopRepository;
        $this->usedFallbackConfigRule = $usedFallbackConfigRule;
        $this->paymentMethodMapper = $paymentMethodMapper;
        $this->paymentMethodWriter = $paymentMethodWriter;
        $this->entityManager = $entityManager;
    }
```